### ISSUE TITLE
Some options from config/CLI ignored or not copied

### ISSUE THREAD

**Issue by keeffEoghan** [KidkArolis/jetpack#34]
Some configuration options (from `jetpack.config.js` or CLI) don't seem to be copied into the `options` passed into Webpack and the rest of the tooling:
- [`options.css`](https://github.com/KidkArolis/jetpack/blob/b6c70874be5affdb68b9325cc84bffa2b417ad4c/lib/cli/options.js#L113): this was where I found the issue; setting `options.css.features['nesting-rules'] = true` wasn't turning on that feature.
- [`options.browsers`](https://github.com/KidkArolis/jetpack/blob/b6c70874be5affdb68b9325cc84bffa2b417ad4c/lib/cli/options.js#L106): also not copied/merged.
- Others: these are the 2 that currently affect me, but other options also seem to be ignored, or can only be set by one of CLI/config file not both (e.g: [`options.hot` seems like it can only be set via CLI, not by config file](https://github.com/KidkArolis/jetpack/blob/b6c70874be5affdb68b9325cc84bffa2b417ad4c/lib/cli/options.js#L103)). It'd be good to be consistent with what options can be set where, and any differences/exceptions noted in the docs.

The relevant part of my `jetpack.config.js` file looks like this:
```javascript
module.exports = {
    // ...
    browsers: [
        'last 2 versions',
        'cover 99%',
        'not < 1%',
        'not dead'
    ],
    css: {
        features: {
            'nesting-rules': true
        }
    }
    // ...
};
```

(Other than that, really liking the tool, saved me some time and setup, and is a good reference approach to learn from!)

**Comment by KidkArolis** [comment]
That was just sloppy of me, I wasn't cleanly merging cli args with file config, in fact I wasn't merging in css/browser features at all, surprised nesting worked for me in my project – perhaps latest browsers support it natively? Thanks for reporting, should be fixed in 0.9.1.

**Comment by keeffEoghan** [comment]
Thanks! Working well now 👍
